common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java

```diff
@@ -697,6 +697,24 @@
   public static final String SUNBIRD_USER_PROFILE_FIELD_DEFAULT_VISIBILITY =
       "sunbird_user_profile_field_default_visibility";
   public static final String DEFAULT_PROFILE_FIELD_VISIBILITY = "defaultProfileFieldVisibility";
+  public static final String COURSE_BATCH_NOTIFICATIONS_ACTIVE = "course_batch_notification_active";
+  public static final String BATCH_END_DATE = "batchEndDate";
+  public static final String BATCH_START_DATE = "batchStartDate";
+  public static final String TEMPLATE_NAME = "templateName";
+  public static final String BATCH_MENTOR_ENROL = "batchMentorEnrol";
```

Comment:
We should not hardcode the template names. The template names should be configurable and that too per tenant.

Reply:
it will be done separately in another task, we will implement it using form api once our templates get inserted into form database